src/build.py

```diff
@@ -332,6 +332,17 @@ def RemoteBranch(branch):
   return WATERFALL_REMOTE + '/' + branch
 
 
+def GitUpdateRemote(src_dir, git_repo, remote_name):
```

Comment:
This really does belong more as a top-level construct. I had felt it odd to have a function that took three arguments where two of which were member variables when it was only used inside that class.
So, nice!